src/frontend/e2e/tests/uploadSample.spec.ts

```diff
@@ -0,0 +1,121 @@
+import {test,expect} from '@playwright/test'
+import {SampleData} from '../utils/schemas/sampleData'
+import {LoginPage} from '../page-objects/LoginPage'
+import {SamplesPage} from '../page-objects/SamplesPage'
+import {UploadPage} from '../page-objects/UploadPage'
+
+
+test.describe("Filter Test Suite",()=>{
+    let loginPage: LoginPage;
+    let samplePage: SamplesPage;
+    let uploadPage: UploadPage;
+
+    test.beforeEach(async ({ page })=>{
+        loginPage = new LoginPage(page);
+        samplePage = new SamplesPage(page);
+        uploadPage = new UploadPage(page);
+        await loginPage.login('lbrambila@contractor.chanzuckerberg.com','Br@mb1la');
```

Comment:
This is NOT good. Pls use the login feature I implemented; you already have this in your branch. Credentials should be in .env and loaded via process.env